test/parallel/test-fs-whatwg-url.js

```diff
@@ -72,3 +72,208 @@ if (common.isWindows) {
     message: `File URL host must be "localhost" or empty on ${os.platform()}`
   }));
 }
+
+// path ending with forward slash are not permitted
+fs.readFile(new URL('file:///c:/tmp/test/'), common.expectsError({
+  code: 'ERR_INVALID_FILE_URL_PATH',
+  type: Error,
+  message: 'File URL path illegal operation on a directory'
+}));
+
+fs.readFile(new URL('file:///tmp/test/'), common.expectsError({
+  code: 'ERR_INVALID_FILE_URL_PATH',
+  type: Error,
+  message: 'File URL path illegal operation on a directory'
+}));
+
+common.expectsError(
+  () => {
+    fs.readFileSync(new URL('file:///c:/tmp/test/'));
+  },
+  {
+    code: 'ERR_INVALID_FILE_URL_PATH',
+    type: Error,
+    message: 'File URL path illegal operation on a directory'
+  });
+
+common.expectsError(
+  () => {
+    fs.readFileSync(new URL('file:///tmp/test/'));
+  },
+  {
+    code: 'ERR_INVALID_FILE_URL_PATH',
+    type: Error,
+    message: 'File URL path illegal operation on a directory'
+  });
+
+//fs.open
```

Comment:
Does the linter pass on this? I think it should complain on comments without a space after `//`?